lightning-invoice/src/utils.rs

```diff
@@ -227,7 +231,7 @@ where
 	let duration = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH)
 		.expect("for the foreseeable future this shouldn't happen");
 	create_invoice_from_channelmanager_and_duration_since_epoch(
-		channelmanager, keys_manager, network, amt_msat, description, duration
+		channelmanager, keys_manager, network, amt_msat, description, duration, invoice_expiry_delta_secs
```

Comment:
I realize we haven't been consistent in this file, but could you wrap parameter and argument lists at 100 characters here and other places that you touched?